src/cray/cfs/operator/events/session_events.py

```diff
@@ -450,6 +530,16 @@ def _get_ansible_container(self, session_data, configuration):
         if session_data["debug_on_failure"]:
             debug_wait_time = options.debug_wait_time
 
+        # Lookup any vault token necessary to decrypt SOPS variables when running ansible
+        try:
+            vault_token_env = client.V1EnvVar(name='VAULT_TOKEN', value=self._lookup_vault_token(session_data) or '')
+        except MultitenantException as mte:
+            LOGGER.warning("Unable to set VAULT_TOKEN for job: %s; skipping, but could cause failed configuration session.",
```

Comment:
```suggestion
            LOGGER.warning("Unable to set VAULT_TOKEN for job. Error: %s; skipping, but could cause failed configuration session.",
```
When I read through this error output, I expected '%s' to be the identifier of the job. Instead, it's just the error output.

Reply:
I agree, but I'd change `Error` to `Reason`. Rahul noticed me using error in a similar context and suggested that we should avoid using error in our log messages. I can see a number of reasons for that, but for me, it seems like it reduces confusion. Especially since here we're declaring this to be a warning, not an error (as far as we're concerned).

Reply:
It's simply a warning and not an error; most people do not yet use kv encryption.